crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -171,3 +206,76 @@ pub struct Capsule {
 }
 impl super::Primitive2d for Capsule {}
 impl Primitive3d for Capsule {}
+
+impl Capsule {
+    /// Create a new `Capsule` from a radius and half-length
+    pub fn new(radius: f32, half_length: f32) -> Self {
+        Self {
+            radius,
+            half_length,
+        }
+    }
+}
+
+/// A cone primitive.
+#[derive(Clone, Copy, Debug)]
+pub struct Cone {
+    /// radius of the base
+    pub radius: f32,
+    /// height of the cone
+    pub height: f32,
+}
+impl Primitive3d for Cone {}
+
+impl Cone {
+    /// Create a new `Cone` from a radius and height
+    pub fn new(radius: f32, height: f32) -> Self {
+        Self { radius, height }
+    }
+}
+
+/// A conical frustum primitive.
+/// A conical frustum can be created
+/// by slicing off a section of a cone.
+#[derive(Clone, Copy, Debug)]
+pub struct ConicalFrustum {
+    /// Radius of the top of the frustum
+    pub radius_top: f32,
+    /// Radius of the base of the frustum
+    pub radius_bottom: f32,
+    /// Height of the frustum
+    pub height: f32,
```

Comment:
Again, don't know but maybe a half-height would be more efficient like for other shapes?

Reply:
Full height here :)